apiserver/facades/client/application/get_test.go

```diff
@@ -132,23 +156,28 @@ var getTests = []struct {
 	expect: params.ApplicationGetResults{
 		Config: map[string]interface{}{
 			"title": map[string]interface{}{
+				"default":     "My Title",
 				"description": "A descriptive title used for the application.",
+				"source":      "default",
 				"type":        "string",
 				"value":       "My Title",
-				"is_default":  true,
 			},
 			"outlook": map[string]interface{}{
 				"description": "No default outlook.",
 				"type":        "string",
+				"source":      "user",
 				"value":       "phlegmatic",
 			},
 			"username": map[string]interface{}{
+				"default":     "admin001",
 				"description": "The name of the initial account (given admin permissions).",
+				"source":      "user",
```

Comment:
It would be good to have a clearer indication in this test that we are explicitly choosing to set the value to the default value, but we still see the source as 'user' because the user explicitly requested it.